AsyncDisplayKit/ASTextNode.mm

```diff
@@ -410,6 +418,8 @@ + (void)drawRect:(CGRect)bounds withParameters:(ASTextNodeDrawParameters *)param
 
 - (NSObject *)drawParametersForAsyncLayer:(_ASDisplayLayer *)layer
 {
+  [self _invalidateRendererIfNeeded];
```

Comment:
@soniccat is correct.  Though it is something I would like to change in the future, for a variety of reasons, it will be a complex change, so I'm not worried about relying on this for now.  My main concern is that this could introduce some other inefficiency or behavior change from what clients have relied on to date, but it may be safe.